homeworks/olga-papina/01-html/home.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+</head>
+
+<body>
+    <div class="topnav">
+        <a class="active" href="home.html">Home</a>
+        <a href="contact.html">Contact</a>
+    </div>
+    <hr />
+    <h1>
+        Hi, I am Olga,<br />
+        Test Automation Engineer
+    </h1>
+    <div>
+        <p>I'm a test automation engineer. I've over a year of experience in this sphere.<br />
+            Before automation I've worked as a manual tester on web and mobile projects.<br />
+            I'm an attentive to details person. I've got a technical mind and great communication skills.
+        </p>
+    </div>
+    <div>
+        <img src="./images/photo_olpapina.png" height="100" width="110" alt="Olga's photo">
+    </div>
+    <div>
+        <a href="#">Download Resume</a>
+    </div>
+    <hr />
+    <footer>
+        <img src="./images/facebook_icon.svg" height="24" width="24" alt="facebook icon">
+        <img src="./images/instagram_icon.svg" height="24" width="24" alt="instagram icon">
+        <img src="./images/linkedin_icon.svg" height="24" width="24" alt="linkedin icon">
+        <img src="./images/tweet_icon.svg" height="24" width="24" alt="tweet icon">
```

Comment:
idem last comment.
Also it would be nice if u add some href to the icons. Just the main page of every social-media..
Remember, you shouldnt have things that do nothing